ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -59,17 +59,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-ovn-ipsec-amd-mixarch-day2-f14
```

Comment:
If I remember right, this job is used for "upgrade a single-arch cluster, then scale a different arch of worker to have a day2 heterogeneous cluster."

Reply:
This job has already removed from openshift-openshift-tests-private-release-4.16__multi-stable.yaml, so I think we don't need to add upgrade job for it as well.

Reply:
```
[jianl@jianl-thinkpadt14gen4 release]$ grep -r 'aws-ipi-ovn-ipsec' ci-operator/config/openshift/openshift-tests-private/openshift-openshift-t
ests-private-release-4.16__multi-stable.yaml
- as: aws-ipi-ovn-ipsec-arm-mixarch-f14
    workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
[jianl@jianl-thinkpadt14gen4 release]$ 
```

Reply:
This job was added for a OTA epics, not a regular upgrade job from e2e ci. cc @shellyyang1989 to confirm.

Reply:
Agree. Let's keep it please.